client.go

```diff
@@ -114,9 +115,13 @@ func newRPC2Client(conn net.Conn) (*OvsdbClient, error) {
 			ovs.Apis[db] = NewNativeAPI(schema)
 		} else {
 			c.Close()
+			errors = append(errors, err.Error())
 			return nil, err
 		}
 	}
+	if len(errors) > 0 {
+		return nil, fmt.Errorf("Multiple DB Errors: %s", strings.Join(errors, ","))
```

Comment:
do we have unit test for this; (may be I missed) as in case of error it will return on L119?

Reply:
You're right. Thanks. Not sure how useful error aggregation is here anyway. It's not very common to have a ovsdbserver with multiple databases. In any case I'll fix it